### PR TITLE
Update to russian locale

### PR DESCRIPTION
**Pull request by IRainman**
Added untranslated strings
Fixed multiple inaccuracies and (or) errors in translation
Corrected the use of the letter 'ё'

P.S. I checked the translation file not completely, later I will continue this work.
P.P.S. I apologize for the fledged formatting.

### CODE REVIEW

File: ru/messages.json [hunk 1]
```diff
@@ -275,7 +285,7 @@
         "message": "Добавить новый скрипт..."
     },
     "Get_new_scripts___": {
-        "message": "Загрузить новые скрипты..."
+        "message": "Получить новые скрипты..."
```
Comment (BychekRU) [inline ru/messages.json]: May be "Добавить"? I think it'll be better

File: ru/messages.json [hunk 2]
```diff
@@ -329,22 +339,25 @@
...
+        "message": "Параметры совместимости с GreaseMonkey/Firefox"
     },
     "Comment": {
         "message": "Комментарий"
     },
     "Details": {
         "message": "Детали"
     },
+    "UUID": {
+        "message": ">>>UUID<<<"
```
Comment (BychekRU) [inline ru/messages.json]: Please delete ">>>" and "<<<" symbols

File: ru/messages.json [hunk 3]
```diff
@@ -383,7 +396,7 @@
         "message": "Вставить конструктор"
     },
     "Auto_Indent_all": {
-        "message": "Автоотступ"
+        "message": "Авто-отступ везде"
```
Comment (BychekRU) [inline ru/messages.json]: "Авто-отступ" is incorrect, use "Автоотступ"

File: ru/messages.json [hunk 4]
```diff
@@ -439,13 +452,13 @@
...
     "Show_backups": {
-        "message": "Показать бекапы"
+        "message": "Показать резервные копии"
     },
     "Delete": {
         "message": "Удалить"
     },
     "Features": {
-        "message": "Разрешения"
+        "message": "Возможности"
```
Comment (BychekRU) [inline ru/messages.json]: This is used near icon saying "This script has access to https pages". I think "Разрешения" is better in this case

File: ru/messages.json [hunk 5]
```diff
@@ -563,7 +579,7 @@
         "message": "Общие"
     },
     "Script_Update": {
-        "message": "Обновление скриптов"
+        "message": "Обновление скрипта"
```
Comment (BychekRU) [inline ru/messages.json]: Left "Обновление скриптов" please, it's used for all scripts not for only selected one

File: ru/messages.json [hunk 6]
```diff
@@ -640,10 +668,10 @@
         "message": "Умный"
     },
     "Line_break": {
-        "message": "Разрыв строк"
+        "message": "Разрыв строки"
     },
     "Reindent_on_typing": {
-        "message": "Выравнивание при вводе"
+        "message": "Выравнивание при наборе текста"
```
Comment (BychekRU) [inline ru/messages.json]: We don't write "текст", we write "код" in this extension, so it's better left just "вводе" or use "наборе кода"

File: ru/messages.json [hunk 7]
```diff
@@ -828,10 +879,10 @@
         "message": "Синтаксических ошибок не найдено."
     },
     "One_error_or_hint_was_found_": {
-        "message": "Найдена одна ошибка или предупреждение."
+        "message": "Найдена ошибка или доступен совет."
```
Comment (BychekRU) [inline ru/messages.json]: It's not "совет". Don't use literal translation

File: ru/messages.json [hunk 8]
```diff
@@ -828,10 +879,10 @@
...
     },
     "0count0_errors_or_hints_were_found_": {
-        "message": "$count$ ошибок или предупреждений найдено.",
+        "message": "$count$ ошибок или советов найдено.",
```
Comment (BychekRU) [inline ru/messages.json]: It's not "совет". Don't use literal translation

**Comment by bovirus** [comment]
@derjanb 

Could you please integrate it?

**Comment by BychekRU** [comment]
Hi @IRainman, if you can't fix my notes and formatting I can do that instead of you

File: ru/messages.json (hunk 1 above)
Comment (IRainman) [reply ru/messages.json]: "Добавить" is really much better.

File: ru/messages.json (hunk 7 above)
Comment (IRainman) [reply ru/messages.json]: Ok, thank you!

File: ru/messages.json [hunk 9]
```diff
@@ -1182,6 +1263,12 @@
     "Error": {
         "message": "Ошибки"
     },
+    "Externals": {
+        "message": "Внешние"
+    },
+    "Requires": {
+        "message": "Необходимая"
```
[the remark this answers is not in the document]
Comment (IRainman) [reply ru/messages.json]: Yes, it's there, a lot of untranslated in the "expert" display mode.

**Comment by IRainman** [comment]
@BychekRU Thank you, yes, please correct the mistakes for me because I do not have enough time now.

P.S. I took up the translation only because it took an emergency transition to Tempermonkey because of the Firefox update to the new engine and, as a consequence, the complete degregation of the most commonly used Firefox extension for scripts.

File: ru/messages.json (hunk 9 above)
[the remark this answers is not in the document]
Comment (IRainman) [reply ru/messages.json]: Yes, it's there. In the "expert" display mode.

**Pull request closed without merging**